app/frontend/i18n/i18n.ts

```diff
@@ -119,11 +119,18 @@ const options = {
           goTo: "Go to {{ location }}",
           permitApp: "permit app",
           adminPanel: "admin panel",
+          enabledCommunitiesDescription: "Communities you can submit Building Permit applications in:",
+          enabledCommunities: [
+            { name: "City of North Vancouver", href: "/jurisdictions/corporation-of-the-city-of-north-vancouver" },
+          ],
```

Comment:
can you move this to constants. Seems awkward to have data structures in the translation file. You can place the name translation here.